### PR TITLE
Us27 order testimonies by validation

### PR DESCRIPTION
**Pull request by Vannou28**

### CODE REVIEW

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -51,4 +51,32 @@ public function addService(): string
...
+
+            // TODO validations (length, format...)
+
+            // if validation is ok, insert and redirection
+            $testimoniesManager = new TestimoniesManager();
+            $id = $testimoniesManager->insert($testimony);
+            header('Location: /Admin/Testimony/show/' . $id);
+        }
+        return $this->twig->render('Admin/Testimony/add.html.twig');
+    }
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php:68]: duplicate of another PR

File: src/Model/TestimoniesManager.php [hunk 2]
```diff
@@ -38,4 +38,12 @@ public function update(array $item): bool
 
         return $statement->execute();
     }
+
+    public function selectValidation(string $validation = 'false'): array
```
Comment (LucyMerlier) [inline src/Model/TestimoniesManager.php:42]: wouldn't it be better with a boolean?

File: src/Controller/AdminController.php [hunk 3]
```diff
@@ -51,4 +51,32 @@ public function addService(): string
...
+
+    public function selectValidation(string $validation): string
+    {
+        $testimoniesManager = new TestimoniesManager();
+        $testimonies = $testimoniesManager->selectValidation($validation);
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php:73]: no form validations?

File: src/Controller/AdminController.php [hunk 4]
```diff
@@ -51,4 +51,32 @@ public function addService(): string
...
+
+        $servicesManager = new ServicesManager();
+        $services = $servicesManager->selectAll();
+        return $this->twig->render('Admin/Home/index.html.twig', [
+            'services' => $services,
+            'testimonies' => $testimonies
+        ]);
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php:80]: so, your admin home page will be named "/admin/selectValidation"?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Pull request closed without merging**